021-IdentityForApps/Coach/02-susi.md

```diff
@@ -0,0 +1,35 @@
+# Challenge 2 - Sign Up, Sign In User Flow
```

Comment:
You'll want this to be "#2" instead of just "2" here to match the other 12 challenge texts.

Reply:
done